.devcontainer/vmr-source-build/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// Container suitable for investigating issues with source build
+// Contains the VMR (dotnet/dotnet) with applied changes from the current PR
+// The container supports source-building the SDK
+{
+    "name": "VMR with PR changes",
+    "image": "mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-36",
+    "hostRequirements": {
+        // A completely built .NET source-tarball is >64 GB
+        "storage": "128gb"
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "ms-dotnettools.csharp",
+                "eamodio.gitlens"
```

Comment:
This extension is a powerful investigative tool but it's probably not as valuable in the VMR now since all the commits are created by dotnet maestro. Plus it is owned by GitKraken now, I think we should remove it.

Reply:
Yeah, makes sense. I removed it from all VMR containers